appointment-svc/router/index.ts

```diff
@@ -35,4 +37,10 @@ router.delete(
   appointmentController.deleteOne
 );
 
+router.post('/appointment/hash', (req, res) => {
+  const { hash, encryptionIV } = req.body
+  const decryptedString = decryptData(hash, encryptionIV)
+  const decryptedData = JSON.parse(decryptedString)
+})
+
```

Comment:
I thought that we decided that we don't need this endpoint because the decryption of the hash will be in middleware, we don't need to share it with the appointment API endpoint. And we will pass encryptionIV as a query to the frontend.